vendor/lancier-monitoring/pipesens-plus-codec.yaml

```diff
@@ -0,0 +1,78 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: sensor.js
+  # Examples (optional)
+  examples:

```

Comment:
this example also throw an error I have to comment all lines, you can do it until the are correct.